html/elements/area.json

```
@@ -520,6 +520,67 @@
             }
           }
         },
+        "ping": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "79"
```

```suggestion
                "version_added": "17"
```

api.HTMLAreaElement.ping says "17" and I can confirm it using http://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement/ping. With my test in https://github.com/mdn/browser-compat-data/pull/9470#issuecomment-817091562 I can also confirm the behavior for both `<a ping` and `<area ping>` was introduced in Edge 17.

---

Fixed with 1878c56dd.